packages/design-tokens/src/color/utils/blendColorTransparency.test.ts

```diff
@@ -0,0 +1,46 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import { colors } from '../colors'
+import { blendColorTransparency } from './blendColorTransparency'
+
+const { text } = colors
+
+describe('blendColorTransparency', () => {
+  test('text - 2', () => {
+    expect(blendColorTransparency(text, 2)).toEqual('rgba(38,45,51,0.12)')
+  })
```

Comment:
```suggestion
describe('blendColorTransparency', () => {
  const alphaLevel = 2
  const targetUiBlend = uiBlends[alphaLevel - 1]
  test('text - 2', () => {
    expect(blendColorTransparency(text, alphaLevel)).toEqual(`rgba(38,45,51,${targetUiBlend}`)
  })
```

Could be helpful to derive the '0.12' in the `toEqual` call using the `uiBlends` array. Otherwise, it sort of feels like '0.12' is a magic number.

My code suggestion though might not be very readable, in which case, maybe a comment explaining where 0.12 comes from would be more helpful!

Reply:
See above. 🤦